### PR TITLE
WLA-442: Global changes for Welsh

### PR DESCRIPTION
**Pull request by LucaDelBuonoHMCTS**

### CODE REVIEW

File: middleware/draftAppealStoreMiddleware.js [hunk 1]
```diff
@@ -124,10 +122,10 @@ class SaveToDraftStoreAddAnother extends AddAnother {
 
   get continueText() {
     if (this.req.idam) {
-      return content.continueButtonText.save;
+      return 'saveAndContinue';
```
Comment (srochani) [inline middleware/draftAppealStoreMiddleware.js:125]: this hardcoded value, can we please refer from json file

File: middleware/draftAppealStoreMiddleware.js [hunk 2]
```diff
@@ -124,10 +122,10 @@ class SaveToDraftStoreAddAnother extends AddAnother {
...
     }
 
-    return content.continueButtonText.continue;
+    return 'continue';
```
Comment (srochani) [inline middleware/draftAppealStoreMiddleware.js:128]: this hardcoded value, can we please refer from json file

File: middleware/draftAppealStoreMiddleware.js [hunk 3]
```diff
@@ -158,10 +156,10 @@ class SaveToDraftStore extends Question {
   }
   get continueText() {
     if (this.req.idam) {
-      return content.continueButtonText.save;
+      return 'saveAndContinue';
```
Comment (srochani) [inline middleware/draftAppealStoreMiddleware.js:159]: same here

File: steps/appeal-form-download/content.en.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
-  "titleHead": "Download appeal form - Appeal a benefit decision - GOV.UK",
+  "titleHead": "Download appeal form",
```
Comment (srochani) [inline steps/appeal-form-download/content.en.json:2]: why this title head change, any perticuler reason

File: steps/appointee/appointee-contact-details/content.en.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "titleHead": "Enter contact details - Appeal a benefit decision - GOV.UK",
+  "titleHead": "Enter contact details",
```
Comment (srochani) [inline steps/appointee/appointee-contact-details/content.en.json:2]: same here

File: steps/appointee/appointee-name/content.en.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
-  "titleHead": "Enter name - Appeal a benefit decision - GOV.UK",
+  "titleHead": "Enter name",
```
Comment (srochani) [inline steps/appointee/appointee-name/content.en.json:2]: same here

File: steps/appointee/appointee-dob/content.en.json [hunk 7]
```diff
@@ -1,5 +1,5 @@
 {
-  "titleHead": "Enter date of birth - Appeal a benefit decision - GOV.UK",
+  "titleHead": "Enter date of birth",
```
Comment (srochani) [inline steps/appointee/appointee-dob/content.en.json:2]: same here

File: steps/appeal-form-download/content.en.json (hunk 4 above)
Comment (LucaDelBuonoHMCTS) [reply steps/appeal-form-download/content.en.json:2]: `windowTitleSuffix` has been added to `commonContent.json` and is now being appended to all templates

File: middleware/draftAppealStoreMiddleware.js (hunk 1 above)
Comment (LucaDelBuonoHMCTS) [reply middleware/draftAppealStoreMiddleware.js:125]: That hardcoded value represents the KEY in the json file for the VALUE to use in the template.
Makes sense?

File: steps/appeal-form-download/content.en.json (hunk 4 above)
Comment (LucaDelBuonoHMCTS) [reply steps/appeal-form-download/content.en.json:2]: https://github.com/hmcts/sscs-submit-your-appeal/blob/master/commonContent.json

**Pull request merged**